packages/api/lib/writeRecords/write-granules.js

```diff
@@ -3,6 +3,7 @@
 const AggregateError = require('aggregate-error');
 const isEmpty = require('lodash/isEmpty');
 const omit = require('lodash/omit');
+const isArray = require('lodash/isArray');
```

Comment:
This file is the most dangerous part of the merge. I think everything looks right though. Were there any conflicts on this latest round that looked strange or that you were unsure about?

Reply:
No, I did not see any new conflicts while reviewing the code this time. Is there anything else that you think should be addressed?